Sources/PubNub/KMM/Wrappers/AnyJSONObjC.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  AnyJSONObjC.swift
+//
+//  Copyright (c) PubNub Inc.
+//  All rights reserved.
+//
+//  This source code is licensed under the license found in the
+//  LICENSE file in the root directory of this source tree.
+//
+
+import Foundation
+
+@objc
+public class AnyJSONObjC : NSObject {
+    let value: AnyJSON
+    public init(_ value: AnyJSON) {
+        self.value = value
+    }
+    
+    @objc
+    public func asString() -> String? {
+        return value.stringOptional
```

Comment:
You can skip the `return` keyword for single-line expression functions that returns a value

Reply:
done